test/cri-containerd/runpodsandbox_test.go

```diff
@@ -1137,6 +1138,93 @@ func Test_RunPodSandbox_Mount_SandboxDir_LCOW(t *testing.T) {
 	//TODO: Parse the output of the exec command to make sure the uvm mount was successful
 }
 
+func Test_RunPodSandbox_CPUGroup_LCOW(t *testing.T) {
+	requireFeatures(t, featureLCOW)
+
+	pullRequiredLcowImages(t, []string{imageLcowK8sPause})
+
+	ctx := context.Background()
+
+	presentID := "FA22A12C-36B3-486D-A3E9-BC526C2B450B"
+	// we believe it is a resonable assumption that our test machines have at least two cores
```

Comment:
Copying from the other PR but we don't have to assume, why not just check here to make sure?

Reply:
Fixed to actually query for the list of indices.

Reply:
Thanks!